addons/mail/models/mail_thread.py

```diff
@@ -1812,11 +1812,14 @@ def message_post(self, *,
         author_info = self._message_compute_author(author_id, email_from, raise_exception=True)
         author_id, email_from = author_info['author_id'], author_info['email_from']
 
+        if subtype_xmlid:
+            # optimize main use cases
+            if subtype_xmlid in ('mail.mt_note', 'mail.mt_comment'):
+                subtype_id, internal = self.env['ir.model.data'].xmlid_to_res_id(subtype_xmlid), subtype_xmlid == 'mail.mt_note'
```

Comment:
What's the point of computing/assigning a value to internal here, it doesn't appear to be used in the method?